### PR TITLE
Test case of chart repo

### PR DESCRIPTION
**Pull request by ninjadq**
Add test case for helm chart

### CODE REVIEW

File: tests/resources/Harbor-Pages/Project-Helmcharts.robot [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    Click Element  xpath=//hbr-helm-chart//a[contains(.,"${chart_name}")]
+    Sleep 2
+    Capture Page Screenshot  viewchartversion.png
+    Wait Until Page Contains  7.0.2
+
+Go Into Chart Detail
+    [Arguments]  ${version_name}
+    Click Element  xpath=//hbr-helm-chart-version//a[contains(.,"${version_name}")]
+    Sleep  2
+    Page Should Contain Element  ${chart_detail}
```
Comment (reasonerjt) [inline tests/resources/Harbor-Pages/Project-Helmcharts.robot]: Keyword like `Go Into xxx` should be reusable, so the verification should be moved to the test case level.

File: tests/resources/Harbor-Pages/Project-Helmcharts.robot [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    Sleep  2
+
+    Wait Until Page Contains  ${prometheus_chart_name}
+
+Go Into Chart Version
+    [Arguments]  ${chart_name}
+    Click Element  xpath=//hbr-helm-chart//a[contains(.,"${chart_name}")]
+    Sleep 2
+    Capture Page Screenshot  viewchartversion.png
+    Wait Until Page Contains  7.0.2
```
Comment (reasonerjt) [inline tests/resources/Harbor-Pages/Project-Helmcharts.robot]: 7.0.2 should not be hard coded?

**Comment by reasonerjt** [comment]
An overall comment is that when you write a Keyword to illustrate an action, it should be re-usable, so you should leverage input param and should not do verification.

File: tests/robot-cases/Group1-Nightly/Common.robot [hunk 3]
```diff
@@ -623,4 +623,34 @@ Test Case - Admin Push Signed Image
...
     Should Be Equal As Integers  ${rc}  0
-    Should Contain  ${output}  sha256
\ No newline at end of file
+    Should Contain  ${output}  sha256
+
+Test Case - List Helm Charts
+    Init Chrome Driver
+    ${d}=   Get Current Date    result_format=%m%s
+
+    Create An New Project With New User  url=${HARBOR_URL}  username=test${d}  email=test${d}@vmware.com  realname=test${d}  newPassword=Test1@34  comment=harbor  projectname=project${d}  public=false
```
Comment (wy65701436) [inline tests/robot-cases/Group1-Nightly/Common.robot]: please use user027

**Pull request merged**